### DISCUSSION TITLE
Inline discount on the total

### DISCUSSION THREAD

**Discussion by almooradi-dev** [andrewdwallo/erpsaas#86] [Ideas]
Hello,

Why not adding the ability to add a discount (percentage/fixed) at the end of the invoice/bill, instead of just add the discounts for the items.

In addition, why adding discounts only using the "adjustments", I think it's better to add custom discount values.

Thank you

**Comment by andrewdwallo** [comment]
It will be added. The latest release is kind of a pre-release. It's still a WIP.

**Reply by almooradi-dev** [reply]
Great! Thanks

If you need any help with this project, I'm eager to contribute. Just let me know how I can help

**Reply by andrewdwallo** [reply]
If you have any other suggestions, I'm open to listening.

**Reply by almooradi-dev** [reply]
I will add my suggestions/ideas in new discussions, to better organize them
Thanks

**Comment by andrewdwallo** [comment]
@almooradi-dev Your suggestion is implemented in the latest release! https://github.com/andrewdwallo/erpsaas/releases/tag/v3.7.1
